src/Basque/Mews.Fiscalizations.Basque.Tests/Bizkaia/BizkaiaDataValidationTests.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Mews.Fiscalizations.Basque.Dto.Bizkaia.Header;
+using Mews.Fiscalizations.Basque.Tests.Bizkaia.Helpers;
+using System.Text.Json;
+
+namespace Mews.Fiscalizations.Basque.Tests.Bizkaia;
+
+[TestFixture]
+public class BizkaiaDataValidationTests
+{
+    [Test]
+    public void Create_BizkaiaData_JsonSerialization_Succeeds()
+    {
+        var bizkaiaData = BizkaiaDataHelper.CreateSampleBizkaiaData();
+
+        Assert.DoesNotThrow(() =>
+        {
+            string serializedData = JsonSerializer.Serialize(bizkaiaData);
+            Assert.IsFalse(string.IsNullOrEmpty(serializedData));
```

Comment:
Would `JsonSerializer.Serialize` return null in this case?
Anyways, i would use `Assert.IsNotEmpty`